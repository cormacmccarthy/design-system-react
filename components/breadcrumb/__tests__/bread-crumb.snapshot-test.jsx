/* eslint-env jest */
import { testDOMandHTML } from '../../../tests/snapshot-helpers';

import { BREADCRUMB } from '../../../utilities/constants';

import BreadcrumbBase from '../__examples__/base';

testDOMandHTML({
	name: '2 Items',
	test,
	Component: BreadcrumbBase,
	ComponentKind: BREADCRUMB
});

testDOMandHTML({
	name: '1 Item',
	test,
	Component: BreadcrumbBase,
	ComponentKind: BREADCRUMB
});
