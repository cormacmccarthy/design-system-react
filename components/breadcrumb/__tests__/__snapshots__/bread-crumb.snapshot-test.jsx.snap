// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Item DOM Snapshot 1`] = `
<nav
  aria-label="Breadcrumbs"
  role="navigation"
>
  <ol
    className="slds-breadcrumb slds-list--horizontal"
  >
    <li
      className="slds-breadcrumb__item slds-text-title--caps"
    >
      <a
        href="javascript:void(0);"
        id="parent-entity"
      >
        Parent Entity
      </a>
    </li>
    <li
      className="slds-breadcrumb__item slds-text-title--caps"
    >
      <a
        href="javascript:void(0);"
      >
        Parent Record Name
      </a>
    </li>
  </ol>
</nav>
`;

exports[`1 Item HTML Snapshot 1`] = `
"<nav role=\\"navigation\\" aria-label=\\"Breadcrumbs\\">
  <ol class=\\"slds-breadcrumb slds-list--horizontal\\">
    <li class=\\"slds-breadcrumb__item slds-text-title--caps\\"><a id=\\"parent-entity\\" href=\\"javascript:void(0);\\">Parent Entity</a></li>
    <li class=\\"slds-breadcrumb__item slds-text-title--caps\\"><a href=\\"javascript:void(0);\\">Parent Record Name</a></li>
  </ol>
</nav>"
`;

exports[`2 Items DOM Snapshot 1`] = `
<nav
  aria-label="Breadcrumbs"
  role="navigation"
>
  <ol
    className="slds-breadcrumb slds-list--horizontal"
  >
    <li
      className="slds-breadcrumb__item slds-text-title--caps"
    >
      <a
        href="javascript:void(0);"
        id="parent-entity"
      >
        Parent Entity
      </a>
    </li>
    <li
      className="slds-breadcrumb__item slds-text-title--caps"
    >
      <a
        href="javascript:void(0);"
      >
        Parent Record Name
      </a>
    </li>
  </ol>
</nav>
`;

exports[`2 Items HTML Snapshot 1`] = `
"<nav role=\\"navigation\\" aria-label=\\"Breadcrumbs\\">
  <ol class=\\"slds-breadcrumb slds-list--horizontal\\">
    <li class=\\"slds-breadcrumb__item slds-text-title--caps\\"><a id=\\"parent-entity\\" href=\\"javascript:void(0);\\">Parent Entity</a></li>
    <li class=\\"slds-breadcrumb__item slds-text-title--caps\\"><a href=\\"javascript:void(0);\\">Parent Record Name</a></li>
  </ol>
</nav>"
`;
